src/App.tsx

```diff
@@ -11,7 +11,17 @@ const Login = lazy(() => import('./pages/LoginPage'));
 const Dashboard = lazy(() => import('./pages/DashBoard'));
 const AddEdit = lazy(() => import('./pages/AddEdit'));
 
+interface Vacation {
+  id: string;
+  vacationType: string;
+  startDate: string;
+  endDate: string;
+  note: string;
+}
+
 const App: FC = () => {
+  const vacations: Vacation[] = [];
```

Comment:
you shouldn't store vacations here. Please use a ContextAPI, localStore, hook or something like redux.

Reply:
done